### PR TITLE
updated queries, aps and tests

### PR DESCRIPTION
**Pull request by Dragos0000**

### CODE REVIEW

**Review by cosletmihai — APPROVED** [review]

File: dqgen/adapters/ap_reader.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 """
     This module is responsible for reading an application profile (for diffing) from a CSV file.
 """
-NA_VALUES = {"class": "", "property": "", "object property": "", "modifiable": 0, "language dependent": 0}
+NA_VALUES = {"class": "", "property": "", "object property": "", "preview property": "", "preview object property": ""}
```
Comment (costezki) [inline dqgen/adapters/ap_reader.py:14]: oh, you decided to do it here? 
I thought you will do that in the template! Whatever works!

File: dqgen/services/__init__.py [hunk 2]
```diff
@@ -83,4 +83,4 @@
                                        "statistics.html": HTML_TEMPLATES.get_template("statistics.jinja2")}
 
 # QUERY_FALLBACK_LANGUAGES = ["en", "fr", "de", "es"]
-QUERY_FALLBACK_LANGUAGES = ["en"]
\ No newline at end of file
+QUERY_FALLBACK_LANGUAGES = ["en", "fr"]
```
Comment (costezki) [inline dqgen/services/__init__.py:86]: why 2? 
in one variant we shall have "en" and in the second variant the four above

File: dqgen/services/queries_generator.py [hunk 3]
```diff
@@ -24,11 +25,13 @@ def generate_class_level_queries(processed_csv_file: pd.DataFrame, output_folder
 
     for cls in processed_csv_file["class"].unique():
         preview_property = processed_csv_file[(processed_csv_file["class"] == cls)]["preview property"].values[0]
+        preview_object_property = processed_csv_file[(processed_csv_file["class"] == cls)]["preview object property"].values[0]
```
Comment (costezki) [inline dqgen/services/queries_generator.py:28]: and if the list is empty >? Will this code break?

File: dqgen/services/queries_generator.py [hunk 4]
```diff
@@ -77,6 +84,7 @@ def generate_from_csv(ap_file_path:pathlib.Path, output_base_dir:pathlib.Path):
     queries_output.mkdir(parents=True, exist_ok=True)
 
     processed_csv_file = read_ap_from_csv(ap_file_path)
+    validate_application_profile(application_profile_df=processed_csv_file)
```
Comment (costezki) [inline dqgen/services/queries_generator.py:87]: is the validation function updated, to reflect the latest structural changes in the AP CSVs?

**Review by costezki — APPROVED** [review]

File: dqgen/adapters/ap_reader.py (hunk 1 above)
Comment (Dragos0000) [reply dqgen/adapters/ap_reader.py:14]: I wll do it in template...this is something else

File: dqgen/services/queries_generator.py (hunk 4 above)
Comment (Dragos0000) [reply dqgen/services/queries_generator.py:87]: yes

**Pull request merged**